terraform-modules/github-runner/shutdown-script.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+# Arguments from metadata
+ACTIONS_USER=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/actions-user -H "Metadata-Flavor: Google")
+VAULT_ADDR=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/vault-server -H "Metadata-Flavor: Google")
+REPO=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/repo -H "Metadata-Flavor: Google")
+GITHUB_PAT_PATH=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/github-pat-secret-path -H "Metadata-Flavor: Google")
```

Comment:
You could factor these a bit, but OK.

Reply:
Yeah, I suspected there would be ways to make this more abstract but I don't think doing so would result in actually more readable or shorter code. At first glance this is super clear about what's going on.